src/macros-by-example.md

```diff
@@ -122,7 +122,9 @@ fragment specifiers are:
   * `block`: a [_BlockExpression_]
   * `stmt`: a [_Statement_] without the trailing semicolon (except for item
     statements that require semicolons)
-  * `pat`: a [_Pattern_]
+  * `pat2015`: a [_PatternNoTopAlt_]
+  * `pat2021`: a [_Pattern_]
+  * `pat`: equivalent to either `pat2015` or `pat2021`, depending on edition (see note below).
```

Comment:
IIUC, `path2015` and `pat2021` are still unstable, correct?  If so, can you remove them? The reference only documents the stable language.